packaging/node/miden-node.service

```diff
@@ -7,12 +7,10 @@ WantedBy=multi-user.target
 
 [Service]
 Type=exec
+Environment="OTEL_SERVICE_NAME=miden-node"
+EnvironmentFile=/lib/systemd/system/miden-node.env
```

Comment:
How do we override these when we deploy?

I'm also still partial to listing them all in this service file, or in an `override.conf` because then they show up for `systemctl cat <service>` but admittedly one can go find the env file here separately.